src/vaadin-time-picker.html

```diff
@@ -188,6 +187,7 @@
 
             /**
              * Set to true to prevent the user from entering invalid input.
+             * @type {boolean}
```

Comment:
We only use `@type {boolean}` for properties that have default value set to `false`. In this case, `@type` annotation can be removed so the default `boolean | null | undefined` would apply.

Reply:
Done